invenio_app_rdm/communities_ui/views/communities.py

```diff
@@ -12,8 +12,8 @@
 from invenio_communities.views.decorators import pass_community
 
 
-@pass_community
-def communities_detail(pid_value, community):
+@pass_community(serialize=True)
+def communities_detail(pid_value, community, community_ui):
```

Comment:
I don't think we need to keep both `community` and `community_ui` here. The first one is enough :)

Reply:
`Community` is needed to check for the permissions and `community_ui` is needed as it comes with the `ui` key with the necessary translations, why do you think is not needed?